platform/javascript/display_server_javascript.cpp

```diff
@@ -45,21 +45,22 @@
 #define DOM_BUTTON_XBUTTON2 4
 
 char DisplayServerJavaScript::canvas_id[256] = { 0 };
+static bool cursor_inside_canvas = true;
 
 DisplayServerJavaScript *DisplayServerJavaScript::get_singleton() {
 	return static_cast<DisplayServerJavaScript *>(DisplayServer::get_singleton());
 }
 
 // Window (canvas)
-static void focus_canvas() {
+void DisplayServerJavaScript::focus_canvas() {
```

Comment:
The commit says "More", but it's "Less", no? ;)
(Also typo in "Where" but that's really nitpicking.)

Reply:
The rest of the commit message should explain it: `Were static methods in cpp file, polluting global namespace`.
I.e. they were static functions, they are now static methods inside the class DisplayServerJavaScript.